README.md

```diff
@@ -33,12 +33,19 @@ See the differences between version 1.x and 2.x in the wiki article [What's diff
 
 ## Getting started
 
+### Setting up the dependency
+
 The first step is to include RxJava 2 into your project, for example, as a Gradle compile dependency:
 
 ```groovy
 compile "io.reactivex.rxjava2:rxjava:2.x.y"
 ```
 
+(Please replace `x` and `y` with the latest version numbers: [![Maven Central](https://maven-badges.herokuapp.com/maven-central/io.reactivex.rxjava2/rxjava/badge.svg)](https://maven-badges.herokuapp.com/maven-central/io.reactivex.rxjava2/rxjava)
```

Comment:
Badge image is not displaying, I think `[` before `!` is not needed

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#images

Reply:
Ah, you want to make it clickable, then something wrong with syntax or my browser (Firefox 59.0.1)

Reply:
It is the same markdown as in the beginning of the readme file where the top badges are located. Sometimes the maven badge simply fails to load. I'm assuming this is due to throttling at the host service where maven-badges lives.